_LowLevelCode/cpp/test/combine_sqw.tests/combine_sqw.test.cpp

```diff
@@ -221,14 +221,15 @@ TEST_F(TestCombineSQW, Check_Expand_Pix_Map) {
   pix_map.init(TEST_FILE_NAME, BIN_POS_IN_FILE, NUM_BINS_IN_FILE, pix_buffer_size, false);
 
   bool end_pix_reached;
-  size_t num_pix0 = pix_map.check_expand_pix_map(511, pix_buffer_size, end_pix_reached);
+  const std::size_t pix_position{ pix_buffer_size - 1 };
```

Comment:
is `pix_position` the `last_pix_index` or `last_pix_position`? `pix_position` could refer to any pixel in the array.